src/engraving/libmscore/stafftype.cpp

```diff
@@ -256,8 +256,8 @@ void StaffType::write(XmlWriter& xml) const
     if (_invisible) {
         xml.tag("invisible", _invisible);
     }
-    if (_color != QColor(Qt::black)) {
-        xml.tag("color", _color);
+    if (_color != mu::draw::Color(mu::draw::black)) {
```

Comment:
I believe most checks on black color should be replaced with checks on isValid
In this case, by default, the color should be invalid, like QColor
```
QColor::QColor()
Constructs an invalid color with the RGB value (0, 0, 0)
```